docs/intro.rst

```diff
@@ -24,3 +24,78 @@ exception dumps.
 
 For more news about Anaconda development and planned features you can follow
 `our blog <https://rhinstaller.wordpress.com>`_.
+
+History
+-------
+
+(The following was contributed by David Cantrell
+<dcantrell@redhat.com> and is probably 80% to 90% accurate.  History
+is hard to keep track of, but this is what I can recall being both in
+the industry at the time but not at Red Hat.)
+
+So the anaconda code base that we all started with began in 1999 or
+so, but it was not the original installer used by Red Hat Linux.  Like
+many distributions at the time, the installation process was hand
+crafted and consisted of a series of steps executed by a collection of
+different programs and tools (shell scripts or Perl scripts or custom
+programs and so on).  I have not found the original source for that
+installer, but I am still hunting.
+
+OK, so anaconda.  Back in the late 1990s there was a huge push of
+money in to Linux companies.  Red Hat itself was founded in 1993 and
+by the late 90s was a large company.  SuSE was another big one.  I
+worked for a company called Walnut Creek CDROM (ftp.cdrom.com) which
+sponsored FreeBSD and Slackware Linux.  Out in Utah in the United
+States there was a company called Caldera that made a distribution
+called OpenLinux (there's actually a lot more history here, Caldera's
+product called Caldera Network Desktop was based on Red Hat Linux and
+LST (Linux Support Team) Power Linux was based on Slackware
+Linux...LST Power Linux became OpenLinux).
+
+Caldera was founded by former Novell employees and the focus was on
+business customers.  Red Hat was still just trying to make a general
+purpose system and the industry as a whole tended to view the main
+competitor to Linux as Windows.  Caldera was trying to make an
+enterprise operating system.  The late 1990s saw the first dedicated
+Linux trade shows.  No longer did Linux companies go to COMDEX or
+Windows World.  We now had the LinuxWorld Conference and Expo.  And I
+was at the first one.  And the second, and third, ... all the way
+until LWCE stopped being an event.  Companies were using LWCE to drop
+huge product announcements.  Well Caldera had an announcement at one
+of these events.
+
+They announced the next version of Caldera OpenLinux and one of its
+big features was a graphical installer.  This was huge.  This was the
+first Linux distribution to feature a graphical installer program.
+They wrote it in C++ (probably?) and it used Qt.  Everyone wanted to
+know how they made it work.  Since installing packages took a long
+time back then, Caldera even put games in the installer.  You could
+play Tetris during installation (side note, Be did that in BeOS too
```

Comment:
Typo `Be` -> `we`.

Reply:
That is not a typo, it is supposed to be Be, or expanded to "Be Inc." which created BeOS
https://en.wikipedia.org/wiki/BeOS
https://en.wikipedia.org/wiki/Be_Inc.